.gitignore

```
@@ -21,4 +21,4 @@ dist-ssr
 *.sln
 *.sw?
 
-.env
\ No newline at end of file
+.env.yarn/
```

These need to be on separate lines to work.

```suggestion
.env
.yarn/
```